src/idom/web/module.py

```diff
@@ -83,6 +88,7 @@ def module_from_template(
     resolve_exports: bool = IDOM_DEBUG_MODE.current,
     resolve_exports_depth: int = 5,
     unmount_before_update: bool = False,
```

Comment:
Would be nice to have control over `symlink=...` up to at this level.

Reply:
Because the contents of these modules are generated from a template there isn't a file to symlink to.